### ISSUE TITLE
Requesting for android app (open tv)

### ISSUE THREAD

**Issue by farhanffaa** [Fredolx/open-tv#84]
Requesting for android app (open tv)

please create same app it is fast there is no any fast app available in the app store.

**Comment by Fredolx** [comment]
It would be great, but I need more funding to get there. Consider donating to the project or purchasing Open TV on the Microsoft Store. If you have done either in the past, I really appreciate your contribution, thank you.

https://github.com/Fredolx/open-tv/discussions/69

**Comment by farhanffaa** [comment]
yes already donated please try to make app or try to create requested
feature in window app please check issues in gitub issue section

**Comment by Fredolx** [comment]
Thank you again for your donation. There's a few features I still want to deliver to Open TV desktop before embarking into Android

**Comment by farhanffaa** [comment]
Waitng for the new features how i will get update for new features

**Comment by Fredolx** [comment]
v1.1.0 is released today.

**Comment by Fredolx** [comment]
This is already planned so I'll close this issue
